spec/files/v2.4.0/building_151.xml

```diff
@@ -72,7 +72,7 @@
               <auc:YearOfLastMajorRemodel>2003</auc:YearOfLastMajorRemodel>
               <auc:Sections>
                 <auc:Section ID="Default_Section">
-                  <auc:OccupancyClassification>Retail</auc:OccupancyClassification>
+                  <auc:OccupancyClassification>Multifamily</auc:OccupancyClassification>
```

Comment:
```suggestion
                  <auc:OccupancyClassification>Retail</auc:OccupancyClassification>
```

Reply:
Reverting this back to original classification.

Reply:
This is important for the test. Prior to the change in `lib/buildingsync/model_articulation/building_section.rb`, a bsync file with OccupancyClassification Multifamily would break the gem. So I wanted to add that test case